eng/pipelines/aggregate-reports.yml

```diff
@@ -39,8 +39,7 @@ jobs:
           version: "$(DotNetCoreSDKVersion)"
 
       - pwsh: |
-          dotnet restore eng/ApiCompat/ApiCompat.csproj
-          dotnet build eng/ApiCompat/ApiCompat.csproj  /t:GeneratePublishDependencyFile /p:PublishDepsFilePath=$(Build.ArtifactStagingDirectory)/reports/azure-sdk.deps.json
+          dotnet build eng/azure-sdk-deps.csproj  /t:GenerateDepsFile /p:PublishDepsFilePath=$(Build.ArtifactStagingDirectory)/reports/azure-sdk.deps.json
```

Comment:
While this will fill the current gap I think we will want to merge this and the Analyze dependency step before this to both generate and compare our references to the powershell dependencies. I'll try to look further into that later next week, as I'm out on Monday.